ios/Classes/NativeBridge/Handlers/BrowserHandler.swift

```diff
@@ -5,8 +5,9 @@ protocol BrowserHandlerProtocol {
     func handleUrl(url: URL, webSignInType: WebSignInType)
 }
 
-protocol BrowserHandlerToRegisterHandlerProtocol: AnyObject {
-    func handleRedirectURL(url: URL?)
+protocol BrowserHandlerToRegisterHandlerProtocol {
+    func handleRedirectURL(url: URL)
```

Comment:
So here it is fine to omit the label cause `url` is already in the name of method. 
`func handleRedirectURL(_ url: URL)`